CHANGELOG.md

```diff
@@ -4,14 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.6.2]
+## [0.6.3]
```

Comment:
production is at 0.6.1, so we can lump all these entries under 0.6.2

Reply:
I think we recently released [0.6.2](https://github.com/ASFHyP3/gis-services/blob/main/CHANGELOG.md#062), so it should be 0.6.3